### PR TITLE
Environment variable to specify the build branch

### PR DESCRIPTION
**Pull request by victorhurdugaci**
Fixes https://github.com/aspnet/Universe/issues/160

cc @pranavkm

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -463,4 +464,17 @@ functions
                         .Replace("\n", "|n")
                         .Replace("]", "|]");
         }
+
+        string GetBuildBranch()
```
Comment (pranavkm) [inline makefile.shade]: At the top of the file:

`default BUILD_BRANCH = '${Environment.GetEnvironmentVariable("BUILD_BRANCH") ?? "dev"}``

Comment (pranavkm) [inline makefile.shade]: See https://github.com/aspnet/Universe/blob/dev/build/_k-standard-goals.shade#L12

**Comment by victorhurdugaci** [comment]
Updated

File: makefile.shade [hunk 2]
```diff
@@ -421,10 +422,10 @@ functions
         void CloneOrUpdate(string repo)
         {
             var repoUrl = gitHubUriPrefix + repo + ".git";
-
+
```
Comment (pranavkm) [inline makefile.shade]: Extra whitespace

**Comment by pranavkm** [comment]
:shipit:

**Comment by pranavkm** [comment]
@victorhurdugaci could you merge this in to dev?

**Comment by victorhurdugaci** [comment]
Merged in https://github.com/aspnet/Universe/commit/93542965215adebda129b913e0d620e50c57e0a1

**Pull request merged**